rca/enquire_to_study/models.py

```diff
@@ -0,0 +1,106 @@
+from django.db import models
+from django_countries.fields import CountryField
+from modelcluster.fields import ParentalKey
+from modelcluster.models import ClusterableModel
+from phonenumber_field.modelfields import PhoneNumberField
+from wagtail.admin.edit_handlers import (
+    FieldPanel,
+    FieldRowPanel,
+    InlinePanel,
+    MultiFieldPanel,
+)
+from wagtail.core.models import Orderable
+from wagtail.snippets.edit_handlers import SnippetChooserPanel
+from wagtail.snippets.models import register_snippet
+
+
+@register_snippet
+class Funding(models.Model):
+    funding = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.funding
```

Comment:
I don't think we need to change the model name itself, but we should make it clearer in the admin what the snippet is used for specifically.

```suggestion
@register_snippet
class Funding(models.Model):
    funding = models.CharField(max_length=255)
    
    class Meta:
        verbose_name='Enquirey form funding option'
        verbose_name_plural='Enquirey form funding options'
        
    def __str__(self):
        return self.funding
```